bigbluebutton-html5/imports/ui/components/common/button/Button.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const Button = ({
+    size, bgColor, textColor, children,
```

Comment:
as discussed use single line or for multi-line each prop should be in new line

Reply:
fixed